lib/log.rb

```diff
@@ -183,16 +183,20 @@ def reset_dedup
       @dup_buffer.reset if @dup_buffer
     end
 
-    private
-    def censor(msg)
+    private def censor(msg)
```

Comment:
```
private

def censor(msg)
end
```
This is more in line with [ruby style guide](https://rubystyle.guide/#consistent-classes)

Reply:
Not mean to say which way is better. Just following the style used in this repo.
Since it's not an key point of this MR, revert the changes there.